infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/RestApiRequest.java

```diff
@@ -75,31 +74,37 @@ public void respondAsync(final SafeFuture<AsyncApiResponse> futureResponse) {
   public void respondOk(final Object response, final CacheLength cacheLength)
       throws JsonProcessingException {
     context.header(Header.CACHE_CONTROL, cacheLength.getHttpHeaderValue());
-    respond(HttpStatusCodes.SC_OK, JSON_CONTENT_TYPE, response);
+    respond(HttpStatusCodes.SC_OK, response);
```

Comment:
static import ok?

Reply:
Do you mean we should use a static import for `SC_OK`?